pos_report_session_summary/views/report_session_summary.xml

```diff
@@ -0,0 +1,145 @@
+<template id="report_session_summary">
+    <t t-call="web.html_container">
+        <t t-foreach="docs" t-as="o">
+            <t t-call="web.external_layout">
+                <div class="page">
+                    <h2>Session Summary: <span t-field="o.name" /></h2>
+
+                    <div class="row mt32 mb32">
+                        <div class="col-auto mw-100 mb-3">
+                            <strong>Responsible</strong>:<br />
+                            <span t-field="o.user_id" />
+                        </div>
+                        <div class="col-auto mw-100 mb-3">
+                            <strong>Point of Sale</strong>:<br />
+                            <span t-field="o.config_id" />
+                        </div>
+                        <div class="col-auto mw-100 mb-3">
+                            <strong>Opening Date</strong>:<br />
+                            <span t-field="o.start_at" />
+                        </div>
+                        <div class="col-auto mw-100 mb-3">
+                            <strong>Closing Date</strong>:<br />
+                            <span t-field="o.stop_at" />
+                        </div>
+                    </div>
+
+                    <h4>Statement Summary</h4>
+
+                    <table class="table table-sm o_main_table">
+                        <thead>
+                            <tr>
+                                <th>
+                                    <strong>Reference</strong>
+                                </th>
+                                <th>
+                                    <strong>Journal</strong>
+                                </th>
+                                <th class="text-right">
+                                    <strong>Starting Balance</strong>
+                                </th>
+                                <th class="text-right">
+                                    <strong>Total Transactions</strong>
+                                </th>
+                                <th class="text-right">
+                                    <strong>Ending Balance</strong>
+                                </th>
+                                <th class="text-right">
+                                    <strong>Difference</strong>
+                                </th>
+                                <th
+                                        class="text-right"
+                                        groups="base.group_multi_currency"
+                                    >
+                                    <strong>Currency</strong>
+                                </th>
+                                <th class="text-right">
+                                    <strong>Status</strong>
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr t-foreach="o.statement_ids" t-as="statement">
```

Comment:
This field not exist anymore
![image](https://github.com/user-attachments/assets/0ee12960-fb77-44bc-b491-7e6df9ecbb61)